### DISCUSSION TITLE
HTTP API: list of components in a category?

### DISCUSSION THREAD

**Discussion by m-aciek** [WeblateOrg/weblate#10061] [Q&A] [answered]
Hi, I'd like to retrieve a list of components in a [category](https://hosted.weblate.org/api/categories/1/) on Hosted Weblate. There's a [components list endpoint](https://hosted.weblate.org/api/component-lists/), but it seems to not be currently tied with categories? What is components list then? Is there a way to define it in the project? Is support on category components queries on the roadmap? Or is there an undocumented way to do fetch them?

Regards, Maciej

**Comment by nijel** [comment, marked as answer]
You can list all components in a project and filter them client side for a category. Adding components endpoint to a category should be easy, please create an issue for that.
